docs/content/what-is-a-run-id.md

```diff
@@ -0,0 +1,20 @@
+---
+id: what-is-a-run-id
+title: What is a Run Id?
+description: A Run Id is a UUID that a Temporal service assigns to each Workflow run.
+tags:
+  - explanation
+---
+
+import RelatedRead from '../components/RelatedRead.js'
+
+A Run Id is a UUID that a Temporal service assigns to each [Workflow](#) run.
+
+- Temporal guarantees that only one [Workflow Execution](#) with a given [Workflow Id](what-is-a-workflow-id) can be open at a time. But after the [Workflow Execution](#) has completed, if allowed by a configured policy, you might be able to re-execute a [Workflow](#) after it has closed or failed by using the same [Workflow Id](what-is-a-workflow-id).
```

Comment:
Couple things. 
1. Let's lose the bullet points here and paragraph it.
2. Put each sentence on a newline.